src/routes/signup/+page.svelte

```diff
@@ -75,6 +91,8 @@
           type="password"
         />
 
+        <input type="hidden" name="g-recaptcha-response" bind:value={recaptchaToken} />
```

Comment:
btw, we set the reCAPTCHA parameter name here! So we could change it to something nicer to access from the backend, like recaptchaResponse. Then we don't need to use that weird syntax in the backend action.

Reply:
true it is a pretty annoying name